.gitignore

```diff
@@ -23,4 +24,4 @@ dist/previewer*
 dist/*~*
 test/mock-server/*.pem
 test/mock-server/frontend/
-coverage/
+coverage/
```

Comment:
Should be re-saved to append a new-line character here

Reply:
I've updated this file a couple of times locally and pushed it with the additional new-line at the end, but I think gitlab is removing it? [(example)](https://github.com/onfido/onfido-sdk-ui/pull/1414/commits/3abecc112a2491f3cd434d9e26dffd04ca43cfb8) Am I missing something?